oaknational/Oak-Web-Application
# feat:1217 Add terms link to download form

## Description

- Added terms link to downloads form

## Issue(s)

Fixes #1217

## How to test

1. Go to download page of the lesson
2. Check the terms link at the bottom of the form
